### DISCUSSION TITLE
API for @import

### DISCUSSION THREAD

**Discussion by sami616** [vanilla-extract-css/vanilla-extract#534] [Q&A]
Hi! I can see an api for fontFace, but I can't see anything for a top level @import statement? My use case is using this to load a Google font.

**Comment by mattcompiles** [comment]
I'm hesitant to support `@import` as it's designed for including other stylesheets into the one you're writing. I'm not sure what the real life use-cases are for this is in vanilla-extract as you're writing your CSS in TS files, which have their own way of importing things. You can use fonts from google fonts without using the `@import` syntax.

Also as a side note, I don't feel like using `@import` in your production CSS is a great idea as it introduces an extra layer of network waterfall for something that could be known at build time.

**Reply by nayaabkhan** [reply]
I can understand the hesitation about `@import`. However, we have a possibly reasonable use-case: we use a paid font from myfonts.net that requires us to make a call to `@import url("//hello.myfonts.net/count/XXXXXX")` for usage count. I can see it sets a cookie. I think this mechanism doesn't consider performance. Unfortunately, this is something out of our control.
